### ISSUE TITLE
FedCM

### ISSUE THREAD

**Issue by marcoscaceres** [WebKit/standards-positions#309]
### Title of the spec

Federated Credential Management API

### URL to the spec

https://fedidcg.github.io/FedCM/

### URL to the spec's repository

https://github.com/fedidcg/FedCM/

### Issue Tracker URL

https://github.com/fedidcg/FedCM/issues/

### Explainer URL

https://github.com/fedidcg/FedCM/blob/main/explainer.md

### TAG Design Review URL

https://github.com/w3ctag/design-reviews/issues/718

### Description

Allows users to login to websites with their federated accounts in a privacy preserving manner.

**Comment by yi-gu** [comment]
Thanks for merging the requests! When we ship new FedCM extensions in Blink in the future, should we link to this thread instead of opening new ones?

Some extra context: here's the [support](https://lists.webkit.org/pipermail/webkit-dev/2022-March/032162.html) we received from webkit-dev before shipping FedCM API in Blink. In addition, Mozilla is ["broadly supportive](https://github.com/mozilla/standards-positions/issues/618#issuecomment-1221964677)" of the FedCM API. They started [prototyping](https://groups.google.com/a/mozilla.org/g/dev-platform/c/ncmUwK1uO98/m/COhPA4ZrAAAJ) in Firefox and have been collaborating with the Chrome team on [a handful of spec PRs](https://github.com/fedidcg/FedCM/pulls?q=is%3Apr+commenter%3Abvandersloot-mozilla+). Edge recently showed their [support](https://groups.google.com/a/chromium.org/g/blink-dev/c/URpYPPH-YQ4/m/3YjQdq9CAQAJ) of the FedCM API as well.

**Comment by marcoscaceres** [comment]
Seems we already gave a position to FedCM on the mailing list:
https://lists.webkit.org/pipermail/webkit-dev/2022-March/032162.html

**Comment by RByers** [comment]
Yeah that's what we've referred to as "WebKit support" in our [Chromium tracking](https://chromestatus.com/feature/6438627087220736). I was just wondering if maybe it would be a good idea to get this onto the standards position list too, to improve visibility?

**Comment by samuelgoto** [comment]
> Mozilla standards-positions issue URL

FWIW, here is a link to Mozilla's Standards position:

https://github.com/mozilla/standards-positions/issues/618#issuecomment-1221964677

It is almost 2 years old (FedCM has evolved a lot since then) so we should probably ask them to refresh their position, but figured it would be helpful at least to have something there.

**Comment by marcoscaceres** [comment]
Sorry, it's difficult for us to review this because the spec has gotten rather chunky (not a bad thing... but it means that it continues to be challenging to find time to dive into all the details). Continuing to try to get to it.

**Comment by samuelgoto** [comment]
No worries! We would be happy to jump on a call to walk you through it if that helps, as well as give you a sense of directionality (e.g. with passkeys, DBSC, the Login Status API and the digital credentials API)

**Comment by marcoscaceres** [comment]
@samuelgoto, that would help a lot. At the same time, I really would need someone like @rmondello and @pascoej and a few folks that know Web Authn well in the WebKit community to provide an opinion (this is a bit outside my purview).

**Comment by samuelgoto** [comment]
I'd also recommend coordinating with @johnwilander.

**Comment by Bug-Reaper** [comment]
Hey all, anybody here have a take on whether or not 3rd party cookies will be extinguished by Q1 2025?

I don't really want to do the dev for "login with ..." if all these implementations get killed by end-of-year. I also don't really want to implement FedCM and only support chrome.

I can't wrap my head around how many websites are gonna have their login/sign-up bricked by 3rd party cookie deprecation and it seems like the runway to make a clean switch to FedCM is gonna be mere months. Am I missing something here?

**Comment by rblanck** [comment]
We want to push this topic. Because it's crucial for publishers, and after our experience, a real case to support publishers and users for an easier web.

We, as a transatlantic publisher (Axel Springer), strongly support the adoption of the Federated Credential Management API (FedCM) across browsers.

Since we launched FedCM, we've seen a tenfold increase in registrations and verified email addresses! This improvement has led to much more stable and predictable login traffic. The Multi IDP Support feature is a game-changer, offering users greater flexibility and choice among their preferred identity providers. We truly believe it provides an even smoother experience than passkeys, making it easier than ever for our users to connect with us.

Cross-browser support is critical for our users and us. Without it, publishers and IdPs face fragmentation and inconsistent user experiences. A standardized implementation across browsers would reduce complexity, strengthen privacy, and improve trust.

If you would like more information, we are happy to share it.

Where is the status for WebKit at the moment of this feature support?
